extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -16,7 +16,16 @@
 
 import {isLayoutSizeDefined} from '../../../src/layout';
 import {user} from '../../../src/log';
+import {
+    installVideoManagerForDoc,
+} from '../../../src/service/video-manager-impl';
+import {VideoEvents} from '../../../src/video-interface';
+import {videoManagerForDoc} from '../../../src/services';
+import {assertAbsoluteHttpOrHttpsUrl} from '../../../src/url';
 
+/**
+ * @implements {../../../src/video-interface.VideoInterface}
```

Comment:
According to [test-example-validation.js](https://github.com/ampproject/amphtml/blob/master/test/integration/test-example-validation.js#L21) there's a todo of using the local validator. Perhaps we should file an issue to use the local validator now that it is open-sourced.

Seems similar to #8859 where a new attribute was added and the integration test had to be skipped after the fact. Honestly I hadn't seen these integration tests till now (cc @Gregable as an fyi).

It may be best to add the integration test after the attribute is in the released validator and that can be tracked in a separate issue (as opposed to what happened with amp-pixel where there are several issues, one to skip then unskip).